### PR TITLE
chore(demo): `Changelog` fix header font + small customization upgrade

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [X] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?
- [X] Documentation content changes

## What is the current behavior?

before

## What is the new behavior?
afterChangelog

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

**Review by splincode — APPROVED** [review]

File: projects/demo/src/modules/info/changelog/changelog.style.less [hunk 1]
```diff
@@ -48,4 +48,22 @@ markdown {
...
+    & h3[id^='feat']:before {
+        content: '🚀';
+    }
+
+    & h3[id^='bug']:before {
+        content: '🐞';
+    }
+
+    & h3[id^='deprecations']:before {
+        content: '❌';
```
Comment (splincode) [inline projects/demo/src/modules/info/changelog/changelog.style.less:61]: please, use [⚠️](https://emojipedia.org/warning/)

Comment (nsbarsukov) [reply projects/demo/src/modules/info/changelog/changelog.style.less:61]: It looks in this way.
Is it ok ?

warning

Comment (splincode) [reply projects/demo/src/modules/info/changelog/changelog.style.less:61]: the red color of the icon scares me like it's a breaking change
but in any case, as you wish

Comment (nsbarsukov) [reply projects/demo/src/modules/info/changelog/changelog.style.less:61]: No problem, I will change

```suggestion
        content: '⚠️';
```

**Review by vladimirpotekhin — APPROVED** [review]

**Review by waterplea — APPROVED** [review]

**Pull request merged**